Sming/third-party/.patches/esp-open-lwip.patch

```diff
@@ -125,6 +142,24 @@ index eaa8dd6..6a08192 100644
  #endif
  #endif
  
+@@ -1823,7 +1823,7 @@
+  * debug messages of certain types.
+  */
+ #ifndef LWIP_DBG_TYPES_ON
+-#define LWIP_DBG_TYPES_ON               LWIP_DBG_OFF
++#define LWIP_DBG_TYPES_ON               LWIP_DBG_ON
```

Comment:
Leave the debugging to be OFF by default. If someone wants to have it ON he/she should still be able to set it by using the ENABLE_LWIPDEBUG=1